Router/start-selenium-grid-router.sh

```diff
@@ -93,6 +93,11 @@ if [ ! -z "$SE_DISABLE_UI" ]; then
   SE_OPTS="$SE_OPTS --disable-ui ${SE_DISABLE_UI}"
 fi
 
+if [ ! -z "$SE_REJECT_UNSUPPORTED_CAPS" ]; then
+  echo "Appending Selenium options: --reject-unsupported-caps ${SE_REJECT_UNSUPPORTED_CAPS}"
+  SE_OPTS="$SE_OPTS --reject-unsupported-caps ${SE_REJECT_UNSUPPORTED_CAPS}"
+fi
+
```

Comment:
I believe this is not needed in the Router. Only Distributor, and roles that contain it (Hub and Standalone).

Reply:
Thank you, let me remove this. Since I misunderstood when looking into the table of mapping configs section and roles, there is a tickmark at Router.
![image](https://github.com/SeleniumHQ/docker-selenium/assets/23253546/03a47098-2553-44ee-8669-b114c3f5dc21)